plugins/notifications-backend/src/service/router.ts

```diff
@@ -86,6 +87,36 @@ export async function createRouter(
       .catch(next);
   });
 
+  router.put('/notifications/read', (request, response, next) => {
+    const { messageID, user, read } = request.query;
```

Comment:
`messageID` to `messageId`

For consistency, we should change `msgid` in the `createNotification` response to the same `messageId`.

Reply:
done